src/examples.md

```diff
@@ -243,7 +243,7 @@ refinement loop. The problem being solved is again the Laplace
 equation $$-\Delta u = 1$$ with homogeneous Dirichlet boundary
 conditions. The problem is solved on a sequence of meshes which
 are locally refined in a conforming (triangles, tetrahedrons)
```

Comment:
Can we also change "tetrahedrons" to "tetrahedra"?  Both are apparently correct but the later would match "hexahedra" later in the sentence.

Reply:
I'm okay with this, but can you change it consistently everywhere (including the code)?